engine/apps/auth_token/migrations/0001_squashed_initial.py

```diff
@@ -48,7 +48,7 @@ class Migration(migrations.Migration):
                 ('digest', models.CharField(max_length=128)),
                 ('created_at', models.DateTimeField(auto_now_add=True)),
                 ('revoked_at', models.DateTimeField(null=True)),
-                ('expire_date', models.DateTimeField(default=apps.auth_token.models.mobile_app_verification_token.get_expire_date)),
+                ('expire_date', models.DateTimeField(default=timezone.now)),
```

Comment:
this would mean the token would expire immediately? this is `get_expire_date`:
```python3
def get_expire_date():
    return timezone.now() + timezone.timedelta(seconds=MOBILE_APP_AUTH_VERIFICATION_TOKEN_TIMEOUT_SECONDS)
```

Reply:
Data from `MobileAppVerificationToken` won't be migrated on purpose, here `timezone.now` is used just to get rid of the reference to `apps.auth_token.models.mobile_app_verification_token.get_expire_date` which is now deleted.